### PR TITLE
Core/Spells: remove incorrect PvP combat state

### PR DESCRIPTION
**Pull request by ghost**
by @P-Kito; retrieved from #15027.

Neither Vampiric Embrace nor Wandering Plague
should keep PvP players in combat state.

Players can't drink for extended periods of time in arena
debuffed with Death Knight's Wandering Plague + diseases
or when the counterpart is buffed with Vampiric Embrace.
Every tick keeps them in combat. This PR should fix the issue.

### CODE REVIEW

**Comment by Appled** [comment]
I can totally confirm this. Very annoying bug and then it is usually win for DK team in arena if I am OOM and I cannot drink because staying in combat.

**Comment by P-Kito** [comment]
Btw: The same goes for Vampiric Embrace (The Shadow Priests' passive healing). 
You can do the same with Vampiric Embrace (the Healing spell) :+1: 
If you like to open another PR:

```
            case 15290: // vampiric embrace
                spellInfo->AttributesEx3 |= SPELL_ATTR3_NO_INITIAL_AGGRO;
                break;
```

**Comment by joschiwald** [comment]
SPELL_ATTR3_NO_DONE_BONUS not needed

and i suggest to merge the Vampiric Embrace with this one, because it is exactly the same thing

File: src/server/game/Spells/SpellMgr.cpp [hunk 1]
```diff
@@ -3235,12 +3235,17 @@ void SpellMgr::LoadSpellInfoCorrections()
                 spellInfo->Effects[EFFECT_1].BasePoints = -6; // -5%
                 break;
             case 50526: // Wandering Plague
+                spellInfo->AttributesEx3 |= SPELL_ATTR3_NO_INITIAL_AGGRO;
```
Comment (MitchesD) [inline src/server/game/Spells/SpellMgr.cpp]: ``cpp
case 50526: // Wandering Plague
case 15290: // Vampiric Embrace
spellInfo->AttributesEx3 |= SPELL_ATTR3_NO_INITIAL_AGGRO;
break;
``

**Pull request merged**